modules/images-cluster-sample-imagestream-import.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+// * openshift_images/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="images-cluster-sample-imagestream-import_{context}"]
+= Configuring periodic importing of Cluster Sample Operator image stream tags
+
+.Procedure
+
+. Fetch all the imagestreams in the `openshift` namespace by running the following command:
++
+[source,terminal]
+----
+oc get imagestreams -nopenshift
+----
+
+. Fetch the tags for every imagestream in the `openshift` namespace by running the following command:
++
+[source, terminal]
+----
+$ oc get is <image-stream-name> -o jsonpath="{range .spec.tags[*]}{.name}{'\t'}{.from.name}{'\n'}{end}" -nopenshift
+----
++
+For example:
++
+[source, terminal]
+----
+$ oc get is ubi8-openjdk-17 -o jsonpath="{range .spec.tags[*]}{.name}{'\t'}{.from.name}{'\n'}{end}" -nopenshift
+----
++
+.Example output
+[source, terminal]
+----
+1.11	registry.access.redhat.com/ubi8/openjdk-17:1.11
+1.12	registry.access.redhat.com/ubi8/openjdk-17:1.12
+----
+
+. Schedule periodic import of images for each tag present in the image stream by running the following command:
++
+[source,terminal]
+----
+$ oc tag <repository/image> <image-stream-name:tag> --scheduled -nopenshift
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc tag registry.access.redhat.com/ubi8/openjdk-17:1.11 ubi8-openjdk-17:1.11 --scheduled -nopenshift
+$ oc tag registry.access.redhat.com/ubi8/openjdk-17:1.12 ubi8-openjdk-17:1.12 --scheduled -nopenshift
+----
++
+This command causes {product-title} to periodically update this particular image stream tag. This period is a cluster-wide setting set to 15 minutes by default.
+
+. Verify the scheduling status of periodic import by running the following command:
++
+[source,terminal]
+----
+oc get imagestream <image-stream-name> -o jsonpath="{range .spec.tags[*]}Tag: {.name}{'\t'}Scheduled: {.importPolicy.scheduled}{'\n'}{end}" -nopenshift
+----
++
+For example:
++
+[source,terminal]
+----
+oc get imagestream ubi8-openjdk-17 -o jsonpath="{range .spec.tags[*]}Tag: {.name}{'\t'}Scheduled: {.importPolicy.scheduled}{'\n'}{end}" -nopenshift
+----
++
+.Example output
+[source,terminal]
+----
+Tag: 1.11	Scheduled: true
+Tag: 1.12	Scheduled: true
+----
+
+. Remove the periodic import by running the following command:
```

Comment:
In the modular docs theory, this step should be in it's own module, as it is a separate task.

Reply:
Thanks for catching this.